invenio_rdm_records/resources/serializers/dublincore/schema.py

```diff
@@ -36,10 +36,6 @@ class DublinCoreSchema(BaseSchema):
     locations = fields.Method("get_locations")
     formats = fields.Method("get_formats")
```

Comment:
Can be changed the same way as you did in marcxml schema? Also for the `sources` field.

Reply:
You are correct, thanks!